Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/AuditServiceImpl.java

```diff
@@ -75,4 +75,10 @@ private Timestamp getTimeStamp() throws ParseException {
         return new Timestamp(date.getTime());
     }
 
+    public String createMockAuditMessage(long id) {
+        Audit audit = new Audit();
+        audit.setMessage("<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><AuditMessage xmlns=\"http://nhinc.services.com/schema/auditmessage\"><EventIdentification EventActionCode=\"C\" EventDateTime=\"2015-12-17T00:01:34.113Z\" EventOutcomeIndicator=\"0\"><EventID code=\"110107\" displayName=\"Import\" codeSystemName=\"DCM\"/><EventTypeCode code=\"ITI-41\" displayName=\"Provide and Register Document Set-b\" codeSystemName=\"IHE Transactions\"/></EventIdentification><ActiveParticipant UserID=\"http://www.w3.org/2005/08/addressing/anonymous\" UserName=\"Karl Skagerberg\" UserIsRequestor=\"true\" NetworkAccessPointID=\"127.0.0.1\" NetworkAccessPointTypeCode=\"2\"><RoleIDCode code=\"110153\" displayName=\"Source\" codeSystemName=\"DCM\"/></ActiveParticipant><ActiveParticipant UserID=\"https://localhost:8181/Gateway/DocumentSubmission/2_0/NhinService/XDRRequest_Service\" AlternativeUserID=\"6636@VIMEHTA-F05773\" UserIsRequestor=\"false\" NetworkAccessPointID=\"localhost\" NetworkAccessPointTypeCode=\"1\"><RoleIDCode code=\"110152\" displayName=\"Destination\" codeSystemName=\"DCM\"/></ActiveParticipant><AuditSourceIdentification AuditEnterpriseSiteID=\"DoD\" AuditSourceID=\"urn:oid:1.1\"/><ParticipantObjectIdentification ParticipantObjectID=\"SELF-5^^^&amp;1.3.6.1.4.1.21367.2005.3.7&amp;ISO\" ParticipantObjectTypeCode=\"1\" ParticipantObjectTypeCodeRole=\"1\"><ParticipantObjectIDTypeCode code=\"2\" displayName=\"Patient Number\" codeSystemName=\"RFC-3881\"/></ParticipantObjectIdentification><ParticipantObjectIdentification ParticipantObjectID=\"1.3.6.1.4.1.21367.2005.3.9999.33\" ParticipantObjectTypeCode=\"2\" ParticipantObjectTypeCodeRole=\"20\"><ParticipantObjectIDTypeCode code=\"urn:uuid:a54d6aa5-d40d-43f9-88c5-b4633d873bdd\" displayName=\"submission set classificationNode\" codeSystemName=\"IHE XDS Metadata\"/></ParticipantObjectIdentification></AuditMessage>");
+
+        return audit.getMessage();
```

Comment:
Do the setters and getters do any manipulation?  Or can we just return the String, without the added call stack costs?